support/kwok/kwok.go

```diff
@@ -122,89 +138,125 @@ func (k *Cluster) Create(args ...string) (string, error) {
 	if !ok {
 		return "", fmt.Errorf("kwok Cluster.Create: cluster %v still not in 'cluster list' after creation: %v", k.name, clusters)
 	}
-	log.V(4).Info("kwok cluster available: ", clusters)
+	klog.V(4).Info("kwok cluster available: ", clusters)
 
-	// Grab kubeconig file for cluster.
 	return k.getKubeconfig()
 }
 
-// GetKubeconfig returns the path of the kubeconfig file
-// associated with this kwok cluster
-func (k *Cluster) GetKubeconfig() string {
-	return k.kubecfgFile
-}
-
-func (k *Cluster) GetKubeCtlContext() string {
-	return fmt.Sprintf("kwok-%s", k.name)
+func (k *Cluster) CreateWithConfig(configFile string) (string, error) {
+	return k.Create("--config", configFile)
 }
 
 func (k *Cluster) Destroy() error {
-	log.V(4).Info("Destroying kwok cluster ", k.name)
-	if err := k.findOrInstallKwok(k.e); err != nil {
+	klog.V(4).Info("Destroying kwok cluster ", k.name)
+	if err := k.findOrInstallKwokCtl(); err != nil {
 		return err
 	}
 
-	p := k.e.RunProc(fmt.Sprintf(`kwokctl delete cluster --name %s`, k.name))
+	p := utils.RunCommand(fmt.Sprintf(`%s delete cluster --name %s`, k.path, k.name))
 	if p.Err() != nil {
 		return fmt.Errorf("kwok: delete cluster failed: %s: %s", p.Err(), p.Result())
 	}
 
-	log.V(4).Info("Removing kubeconfig file ", k.kubecfgFile)
+	klog.V(4).Info("Removing kubeconfig file ", k.kubecfgFile)
 	if err := os.RemoveAll(k.kubecfgFile); err != nil {
 		return fmt.Errorf("kwok: remove kubefconfig failed: %w", err)
 	}
 	return nil
 }
 
-func (k *Cluster) findOrInstallKwok(e *gexe.Echo) error {
-	if k.version != "" {
-		kwokVersion = k.version
+func (k *Cluster) ExportLogs(dest string) error {
+	if err := k.findOrInstallKwokCtl(); err != nil {
+		return err
 	}
-	// installing kwok means installing kwokctl binary
-
-	if e.Prog().Avail("kwokctl") == "" {
-		log.V(4).Info(`kwokctl not found, installing version @%s`, kwokVersion)
-		if err := k.installKwokCtl(e); err != nil {
-			return err
+	for _, component := range []string{"audit", "etcd", "kube-apiserver", "kube-controller-manager", "kube-scheduler", "kwok-controller", "prometheus"} {
```

Comment:
@reetasingh  PTAL, this is the closest to export log that I could find with the `kwokctl`.

Reply:
Actually, there might be a better way to do this with https://github.com/kubernetes-sigs/kwok/pull/630/files

Let me see if we can find a way to use that

Reply:
@harshanarayana does the code pulls/install kwokctl bin similar to kind ?

Reply:
@vladimirvivien  Yes it does. You can also pass a custom bin file to it from non standard path just like https://github.com/kubernetes-sigs/e2e-framework/pull/287

Reply:
@harshanarayana I think we can say the supported version is v0.3.0 and use `kwokctl export logs`